### DISCUSSION TITLE
history filter not working

### DISCUSSION THREAD

**Discussion by csecsi** [StackStorm/st2#5438] [UI]
Is it just for me that the filter function on the history page does nothing?

**Comment by minsis** [comment]
Can you provide some more information than a single sentence of "it doesn't work"?

Can you provide screenshots?
Can you provide console log output when navigating to history page?
Do you have some weird Ad Blocker turned on?
Do you have ST2_URL's set correctly in configs?

**Comment by csecsi** [comment]
I've already opened an issue ticket and looks like it is really not working: https://github.com/StackStorm/st2web/issues/925
